lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -119,12 +119,17 @@ def execute(request, params)
           require params[:server_addon_path]
           ServerAddon.finalize_registrations!(@stdout)
         when "server_addon/delegate"
-          server_addon_name = params.delete(:server_addon_name)
```

Comment:
We need to avoid deleting the params here since they may be needed in the error message.